src/main/kotlin/com/terraformation/backend/accelerator/api/CohortsController.kt

```diff
@@ -100,6 +106,34 @@ class CohortsController(
     return getCohort(cohortId)
   }
 
+  @ApiResponse200
+  @ApiResponse404
+  @GetMapping("/{cohortId}/modules")
```

Comment:
This removes some flexible querying for cohort modules, but FE always has cohort ID anyways so I think we prefer to keep API usage consistent instead.